cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -175,7 +179,11 @@ if (NOT THRIFT_FOUND)
   set(THRIFT_HOME "${THRIFT_PREFIX}")
   set(THRIFT_INCLUDE_DIR "${THRIFT_PREFIX}/include")
   IF (${UPPERCASE_BUILD_TYPE} STREQUAL "DEBUG")
-    set(THRIFT_STATIC_LIB "${THRIFT_PREFIX}/lib/libthriftd.a")
+    IF (MSVC)
+      set(THRIFT_STATIC_LIB "${THRIFT_PREFIX}/lib/thriftmdd.lib")
```

Comment:
No need here for `if(MSVC)`, we should be able to handle this better by using `CMAKE_STATIC_LIBRARY_PREFIX` and `CMAKE_SHARED_LIBRARY_SUFFIX`